frostsnap_core/Cargo.toml

```diff
@@ -11,6 +11,7 @@ schnorr_fun = {version = "0.10", features = ["bincode", "serde","alloc", "share_
 rand_chacha = { workspace = true }
 sha2.workspace = true
 chacha20 = { version = "0.9", default-features = false }
+chacha20poly1305 = { version = "0.10", default-features = false }
```

Comment:
This is for doing authenticated encryption.

Reply:
We can get this from rust-bitcoin now/soon
https://github.com/rust-bitcoin/rust-bitcoin/pull/2960